File: db4on/core/src/core/com/db4o/nativequery/expr/AndExpression.cs
namespace com.db4o.nativequery.expr
{
	public class AndExpression : com.db4o.nativequery.expr.BinaryExpression
	{
		public AndExpression(com.db4o.nativequery.expr.Expression left, com.db4o.nativequery.expr.Expression
			 right) : base(left, right)
		{
		}

		public override string ToString()
		{
			return "(" + _left + ")&&(" + _right + ")";
		}

		public override void Accept(com.db4o.nativequery.expr.ExpressionVisitor visitor)
		{
			visitor.Visit(this);
		}
	}
}

File: db4onqopt/core/src/com/db4o/nativequery/expr/AndExpression.java
/* Copyright (C) 2004 - 2006  db4objects Inc.  http://www.db4o.com */

package com.db4o.nativequery.expr;


public class AndExpression extends BinaryExpression {
	public AndExpression(Expression left, Expression right) {
		super(left, right);
	}

	public String toString() {
		return "("+_left+")&&("+_right+")";
	}

	public void accept(ExpressionVisitor visitor) {
		visitor.visit(this);
	}
}
